coverage/.resultset.json

```diff
@@ -0,0 +1,205 @@
+{
```

Comment:
This `coverage` folder shouldn't be checked into source control. Please add to gitignore and do a `git rm`